### PR TITLE
permission issue fixed

### PR DESCRIPTION
**Pull request by adithyadinesh0412**

### CODE REVIEW

File: src/services/permissions.js [hunk 1]
```diff
@@ -66,7 +66,11 @@ module.exports = class PermissionsHelper {
 			const filter = { id }
 			const permissions = await permissionsQueries.findPermissionById(id)
 			if (!permissions) {
-				throw new Error('PERMISSION_NOT_FOUND')
+				return responses.failureResponse({
```
Comment (priyanka-TL) [inline src/services/permissions.js:69]: @adithyadinesh0412 just do this return it catch and throw the message from here

File: src/services/permissions.js [hunk 2]
```diff
@@ -88,7 +88,11 @@ module.exports = class PermissionsHelper {
 				})
 			}
 		} catch (error) {
-			throw error
+			return responses.failureResponse({
+				message: error.message,
```
Comment (priyanka-TL) [inline src/services/permissions.js]: @adithyadinesh0412  can u please add message: error.message || error

**Review by priyanka-TL — APPROVED** [review]

File: src/services/permissions.js [hunk 3]
```diff
@@ -88,7 +88,11 @@ module.exports = class PermissionsHelper {
 				})
 			}
 		} catch (error) {
-			throw error
+			return responses.failureResponse({
```
Comment (aks30) [inline src/services/permissions.js]: @adithyadinesh0412 what if DB is down, will you still give bad request?

**Review by aks30 — CHANGES_REQUESTED** [review]
Reviewed on 15 Nov

File: src/services/permissions.js (hunk 3 above)
Comment (aks30) [reply src/services/permissions.js]: We should have different response based on actual issue, bad request only if frontend is giving invalid data

Comment (adithyadinesh0412) [reply src/services/permissions.js]: Resolved this. 
sending back error response if permission is not found.
For any other issues , catch block with send out generic error "Oops something went wrong" with 500 status code

**Review by priyanka-TL — APPROVED** [review]
Approved on Nov 19

**Pull request merged**